From 7dbc85b1309e5d8fd4863bfd0c0851d6fd2d5570 Mon Sep 17 00:00:00 2001
From: Joyce Ooi <joyce.ooi@intel.com>
Date: Wed, 21 Oct 2020 16:03:01 +0800
Subject: [PATCH 095/120] HSD #1508459581: ARM: dts: adjust the clock frequency
 and falling time for i2c1

commit 3eda4f9e3b26efc026ff43deacb1d0ca9057e2ed from
https://github.com/altera-opensource/linux-socfpga.git

This patch adjusts the clock frequency and falling time for i2c1 as the
A10 display port does not work at 100kHz.

Signed-off-by: Joyce Ooi <joyce.ooi@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
index 1ff37b89b109..a65fe4077049 100644
--- a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
@@ -135,9 +135,9 @@
 	 * adjust the falling times to decrease the i2c frequency to 50Khz
 	 * because the LCD module does not work at the standard 100Khz
 	 */
-	clock-frequency = <100000>;
-	i2c-sda-falling-time-ns = <6000>;
-	i2c-scl-falling-time-ns = <6000>;
+	clock-frequency = <50000>;
+	i2c-sda-falling-time-ns = <60000>;
+	i2c-scl-falling-time-ns = <60000>;
 
 	adc@14 {
 		compatible = "lltc,ltc2497";
-- 
2.31.1

